### DISCUSSION TITLE
Error using evrardjp.keepalived role

### DISCUSSION THREAD

**Discussion by asantos-vk** [evrardjp/ansible-keepalived#249] [Q&A] [answered]
Recently i have started my journey implementing magmacore into a baremetal server. It happens that when I use the haproxy playbook, that uses the evrardjp.keepalived role, I receive the following error:

ERROR! this task 'ansible.builtin.command' has extra params, which is only allowed in the following modules: win_command, command, include, include_tasks, add_host, meta, import_tasks, include_vars, include_role, group_by, set_fact, shell, import_role, win_shell, script, raw

The error appears to be in '/root/magma/orc8r/cloud/deploy/bare-metal-ansible/roles/evrardjp.keepalived/tasks/main.yml': line 58, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

# First line of stderr is similar to 'keepalived v1.0.4 (date)'
- name: Output keepalived version
  ^ here

Can someone help me with this?

**Comment by evrardjp** [comment, marked as answer]
Hello.
First, I didn't know that magma was using this role! Good surprise!

FYI, this role was indeed modified to include a test of version to validate the configuration.

There are multiple ways to solve this:
1) First, maybe work with magma folks to ensure that the right version of the role is used and tested. If necessary, temporarily rollback to a previous version of the role.
2) If I could know more about the environment it could help me. I would need to know at least the role's version/sha. (edit: And also a run, showing the failure, and the ansible version).

**Comment by asantos-vk** [comment]
Hi.

Thanks! Here are the role and ansible env version:
evrardjp.keepalived 7.0.0
ansible==2.9.6
jinja2==2.11.1
netaddr==0.7.19
pbr==5.4.4
jmespath==0.9.5
ruamel.yaml==0.16.10

**Reply by asantos-vk** [reply]
I have downgraded evrardjp. keepalived to version 6.1.0, and it worked!
